src/generators/Minecraft.res

```diff
@@ -183,7 +183,7 @@ module MinecraftCharacterLegacy = {
   }
 }
 
-module MinecraftCharacter = {
+module Character = {
```

Comment:
Nice. I'm hoping over time we could add more Minecraft mobs, and might be worth moving into their own module, but this is good for now.

Reply:
> [1] I can see the test with the 33 pixel scaling is glitching a bit. It might be a bug in the internal generator code, not the cuboid code. Can you take a look?

I looked, and this was caused by how Context2d.translate would round up odd halves as it sent them as ints- This should now be fixed.

> [2] I'm guessing the direction argument is to help with organising parts on the page?

The direction argument and the center face argument are to allow for versatility - Most cuboids should be drawn with the default right direction and the front face at the center, but there are a few reasons to allow for other layouts:

- For parts like left arms and legs where the parts are mirrored for symmetry's sake
- For certain bendables, or otherwise when tabs or joints need to be put in specific places
- Having a different center face is very, very useful for saving space with longer cuboids, such as the horse's body. if the body there had to have the front face as the center face, it would not have fit on the page, and the horse generator wouldn't be nearly as compact as it is

Rotations, and reflections, both also have other, more obvious merits- Those that can also be used in conjunction with `direction` and a center face parameter to allow for a lot of options, as I explained in a comment. But I think that adding them we can save for a different PR, since it would significantly expand the function's length and require a lot more testing.